llap-server/src/java/org/apache/hadoop/hive/llap/metrics/LlapDaemonExecutorInfo.java

```diff
@@ -28,7 +28,9 @@
 public enum LlapDaemonExecutorInfo implements MetricsInfo {
   ExecutorMetrics("Llap daemon cache related metrics"),
   ExecutorMaxFreeSlots("Sum of wait queue size and number of executors"),
-  ExecutorNumExecutorsPerInstance("Total number of executor threads per node"),
+  ExecutorNumExecutors("Total number of executor threads"),
+  ExecutorMaxConfiguredFreeSlots("Sum of the configured wait queue size and the configured number of executors"),
```

Comment:
nit: I would name it ExecutorMaxFreeSlotsConfigured

Reply:
Renamed.